.travis.yml

```diff
@@ -31,6 +31,7 @@ matrix:
 
     - env: Cmd='make lz4install && make -C tests test-lz4'
 
+    - env: Cmd='bash tests/partialTests.sh'
```

Comment:
nit-picking here,
but `partialTests` is not a great name.
This could be easily confused.

Let's be a bit more specific.
Example : `conditional_libzstd_builds.sh`
Feel free to propose a better name if you like